support-frontend/assets/pages/paper-subscription-checkout/components/checkoutForm.jsx

```diff
@@ -204,14 +210,21 @@ function CheckoutForm(props: PropTypes) {
               </SelectWithError>
             </FormSection>
             <FormSection title="When would you like your subscription to start?">
-              <InputWithError
-                id="start-date"
-                label="Start date"
-                type="date"
-                error={firstError('startDate', props.formErrors)}
-                value={props.startDate}
-                setValue={props.setStartDate}
-              />
+              <Error htmlFor={null} error={firstError('startDate', props.formErrors)}>
```

Comment:
we don't have a good component to list checkboxes in so in the same pr i implement the chonky boys I'm gonna work on that component too